content/docs/examples/bookinfo/index.md

```diff
@@ -66,42 +66,38 @@ To start the application, follow the instructions corresponding to your Istio ru
 If you use GKE, please ensure your cluster has at least 4 standard GKE nodes. If you use Minikube, please ensure you have at least 4GB RAM.
 {{< /tip >}}
 
-1. Change directory to the root of the Istio installation directory.
-
-1.  Bring up the application containers:
-
-    *   If you are using [manual sidecar injection](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection),
-        use the following command
-
-        {{< text bash >}}
-        $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo.yaml@)
-        {{< /text >}}
-
-        The `istioctl kube-inject` command is used to manually modify the `bookinfo.yaml`
-        file before creating the deployments as documented [here](/docs/reference/commands/istioctl/#istioctl-kube-inject).
+1.  Change directory to the root of the Istio installation directory.
 
-    *   If you are using a cluster with
-        [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection)
-        enabled, label the `default` namespace with `istio-injection=enabled`
+1.  The default Istio installation uses [automatic sidecar injection](/docs/setup/kubernetes/sidecar-injection/#automatic-sidecar-injection).
+    Label the namespace that will host the application with `istio-injection=enabled`:
 
-        {{< text bash >}}
-        $ kubectl label namespace default istio-injection=enabled
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl label namespace default istio-injection=enabled
+    {{< /text >}}
 
-        Then simply deploy the services using `kubectl`
+1.  Deploy your application using `kubectl` as usual:
 
-        {{< text bash >}}
-        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@
-        {{< /text >}}
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo.yaml@
+    {{< /text >}}
 
-    Either of the above commands launches all four microservices as illustrated in the above diagram.
+    The above command launches all four microservices as illustrated in the above diagram.
     All 3 versions of the reviews service, v1, v2, and v3, are started.
 
     {{< tip >}}
     In a realistic deployment, new versions of a microservice are deployed
     over time instead of deploying all versions simultaneously.
     {{< /tip >}}
 
+    *Note: If you disabled automatic sidecar injection during installation and rely on [manual sidecar injection]
```

Comment:
```suggestion
    {{ <tip> }}
    If you disabled automatic sidecar injection during installation and rely on [manual sidecar injection]
```

Reply:
Do not overuse callouts. There is no reason for this to be a note, tip or similar. Make it a simple paragraph adding context. A simple sentence providing the link is sufficient.
```suggestion
    If you disabled automatic sidecar injection during installation, visit our [manual sidecar injection guide](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection) for more information.
```